is_clang script is python2 which has been removed from RP

--- gn/BUILDCONFIG.gn.orig	2020-03-19 21:10:56 UTC
+++ gn/BUILDCONFIG.gn
@@ -79,16 +79,7 @@ if (current_cpu == "") {
   current_cpu = target_cpu
 }
 
-is_clang = is_android || is_ios || is_mac || is_fuchsia ||
-           (cc == "clang" && cxx == "clang++") || clang_win != ""
-if (!is_clang && !is_win) {
-  is_clang = exec_script("//gn/is_clang.py",
-                         [
-                           cc,
-                           cxx,
-                         ],
-                         "value")
-}
+is_clang = false
 
 if (is_android) {
   ndk_host = ""
